global/meta/models.yml

```diff
@@ -315,20 +315,6 @@ user:
     to: vote/user_id
     restriction_mode: A
 
-  # All foreign keys are meeting-specific:
-  # - Keys are smaller (Space is in O(n^2) for n keys
-  #   in the relation), so this saves storagespace
-  # - This makes quering things like this possible:
-  #   "Give me all groups for User X in Meeting Y" without
-  #   the need to get all groups and filter them for the meeting
-  group_$_ids:
-    type: template
-    replacement_collection: meeting
-    fields:
-      type: relation-list
-      to: group/user_ids
-    restriction_mode: A
-
   meeting_ids:
     type: number[]
     description: Calculated. All ids from group_$_ids as integers.
```

Comment:
The field `group_$_ids` does not exist anymore. The description has to be updated.

Is this field necessary at all? It should be quite easy to use user/user_meeting_ids/meeting_id

Reply:
> The field `group_$_ids` does not exist anymore. The description has to be updated.
> 
> Is this field necessary at all? It should be quite easy to use user/user_meeting_ids/meeting_id

No, because there is still no mechanism to delete the user_meeting instance automatically, if the user isn't in some group of the meeting anymore. At the opposite we had to reactivate the meeting_user, if the user gains rights in this meeting again.

Reply:
But please change at least the description of the field.